src/main/java/net/glowstone/command/TitleCommand.java

```diff
@@ -129,9 +126,60 @@ public boolean execute(CommandSender sender, String commandLabel, String[] args)
                 component = convertJson((JSONObject) parsed);
             }
 
-            player.sendTitle(new Title(component));
+            ((GlowPlayer) player).updateTitle(TitleMessage.Action.TITLE, component);
+            ((GlowPlayer) player).sendTitle();
 
             sender.sendMessage("Updated " + player.getName() + "'s title");
+        } else if (action.equalsIgnoreCase("subtitle")) {
+            if (args.length < 3) {
+                sender.sendMessage(ChatColor.RED + "Usage: /title <player> " + action + " <raw json>");
+                return false;
+            }
+
+            StringBuilder message = new StringBuilder();
+
+            for (int i = 2; i < args.length; i++) {
+                if (i > 2) message.append(" ");
+                message.append(args[i]);
+            }
+
+            String raw = message.toString().trim();
+            if (!validJson(raw)) {
+                sender.sendMessage(ChatColor.RED + "Invalid JSON: Could not parse, invalid format?");
+                return false;
+            }
+
+            String component = raw;
```

Comment:
I would appreciate it if you could use Gson instead.

See `JsonObject` and `JsonParser`.